modules/programs/vscode.nix

```
@@ -135,14 +135,24 @@ in {
           or by Visual Studio Code.
         '';
       };
+
+      mutableUserSettings = mkOption {
+        type = types.bool;
+        default = false;
+        example = false;
+        description = ''
+          Whether user settings can be changed manually or by Visual
+          Studio Code.
+        '';
+      };
```

I think the best option is to just use `mkEnableOption`

```suggestion
      mutableUserSettings = mkEnableOption "changing user settings in Visual Studio Code (rather than through Home Manager)";
      };
```